src/FieldWarning/Assets/Units/VoiceComponent.cs

```diff
@@ -12,34 +12,49 @@
  */
 
 using UnityEngine;
+using System.Collections.Generic;
 
 namespace PFW.Units
 {
     public class VoiceComponent : MonoBehaviour
     {
         [SerializeField]
-        private AudioClip _selectAudio, _moveAudio, _attackAudio;
+        private List<AudioClip>  _selectAudio, _moveAudio, _attackAudio; 
```

Comment:
This kind of change makes the existing prefabs drop the contents of the affected fields - in this case the audio clips. Which is fine but you have to update them after so they work again. I think you forgot to push the new prefab versions?